vars/puppetize_vars_4.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+puppet_etc_dir: "/etc/puppetlabs/puppet"
+puppet_conf_source: "puppet4.conf.j2"
+puppet_agent_bin_path: "/opt/puppetlabs/bin/puppet"
+puppet_cert_request_path: "/opt/puppetlabs/puppet/cache/ssl/certificate_requests/"
```

Comment:
Didn't have such a directory on a polte puppetmaster - could you double check this?

It might be e.g. /etc/puppetlabs/puppet/ssl/certificate_requests/

Reply:
Though I now see this variable isn't used (yet?). Ok we could merge this, but would still like to know the intention and what to do about the missing path going forward :)

Reply:
Let's use it! Made some updates to use the path - which also was a reason why the role was for a short while needlessly running puppet agent -t --noop